admin/services/new-check-window-add.service.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const newCheckWindowAddService = {}
+const newCheckWindowAddValidator = require('../lib/validator/new-check-window/new-check-window-add-validator')
+const monitor = require('../helpers/monitor')
+
+/**
+ * Processes request data on adding
+ * @param {Object} requestData
+ */
+newCheckWindowAddService.process = async (requestData) => {
+  const validationError = newCheckWindowAddValidator.validate(requestData)
+  if (validationError.hasError()) {
+    throw validationError
+  }
```

Comment:
would i be correct in assuming that you have yet to implement persistence?

Reply:
yes that's exactly what i meant.  persistence to the database.

Reply:
Thanks for clarifying. 👍